resources/frameworks.yaml

```diff
@@ -113,6 +113,10 @@ TPOT:
 #    population_size: 25
 #    verbosity: 2
 
+mlr3automl:
+  version: '0.0.0.9000'
```

Comment:
version should now be set to `stable` in this file.

Also, please add your framework in lexicographic order to make it easier to find